shared/src/commonMain/kotlin/com/devil/phoenixproject/presentation/screen/MoveToGroupDialog.kt

```
@@ -0,0 +1,197 @@
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import com.devil.phoenixproject.domain.model.RoutineGroup
+
+/**
+ * Dialog for moving routine(s) to a group.
+ * Shows existing groups, a "New Group..." option, and "Remove from Group" option.
+ * Follows the ProfilePickerDialog visual pattern.
+ */
+@Composable
+fun MoveToGroupDialog(
```

detekt (1.23.8)

[warning] The function MoveToGroupDialog appears to be too complex based on Cognitive Complexity (complexity: 33). Defined complexity threshold for methods is set to '25'

(detekt.complexity.CognitiveComplexMethod)